.github/workflows/deploy.yml

```diff
@@ -0,0 +1,44 @@
+name: Deploy
+
+on:
+  release:
+    types: [published]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+    - name: Install apm (with Atom)
+      env:
+        TEMP_DEB: atom-amd64.deb
+      run: |
+        wget -nv -O "$TEMP_DEB" https://atom.io/download/deb &&
+        sudo apt -qq install "$TEMP_DEB"
```

Comment:
Out of interest does Atom not register their package with debian for direct installation?

Reply:
I don't think so? There's another method they give here first: https://flight-manual.atom.io/getting-started/sections/installing-atom/#debian-and-ubuntu-debapt but it looks more complicated to me

Reply:
Strange choice for Atom but fair enough.